tests/test_helpers.py

```diff
@@ -166,7 +166,8 @@ def write_designspace_and_UFOs(designspace, path):
         ufo_path = os.path.join(os.path.dirname(path), basename)
         source.filename = basename
         source.path = ufo_path
-        source.font.save(ufo_path, formatVersion=3)
+        if not (hasattr(source, "layerName") and source.layerName is not None):
```

Comment:
I suggest we bump minimum required fonttools and assume layerName attribute will be there (None by default).
I think it was added with 3.26